docs/.vuepress/config.js

```diff
@@ -76,7 +76,16 @@ module.exports = {
       "/": [
         ["/getting-started/", "Getting Started", 1],
         ["/config/", "Configuration", 0],
-        ["/hardhat-network/", "Hardhat Network", 0],
+        {
+          title: "Hardhat Network",
+          url: "/hardhat-network/",
+          collapsable: false,
+          depth: 1,
+          children: [
+            ["/hardhat-network/", "Explanation", 0],
```

Comment:
Maybe something like "What is it?"

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1657/commits/3383747fecd278d79aa571eb2f06d7fa63997b98